mama/c_cpp/src/c/bridge/qpid/codec.c

```diff
@@ -41,7 +42,7 @@ qpidBridgeMsgCodec_pack (msgBridge      bridgeMessage,
 {
     pn_data_t*          properties      = NULL;
     pn_data_t*          body            = NULL;
-    mamaPayloadType     payloadType     = MAMA_PAYLOAD_UNKNOWN;
+    char                payloadType     = MAMA_PAYLOAD_UNKNOWN;
```

Comment:
changed to MAMA_PAYLOAD_ID_UNKNOWN in (8171ec826d37f9f99b661069ad24b1928b97f8ee)